modules/ROOT/pages/workday/workday-connector-examples.adoc

```diff
@@ -45,34 +44,30 @@ ns ns0 urn:com.workday/bsvc
 }
 ----
 +
-//image::workday/tranform-dataweave-add-fund-example.png[Transform dataweave for add fund request]
-+
-. Add the Workday connector. Select the web service *Financial Management*, then configure, test, and save a connection.
-+
-//image::workday/workday-configuration-design-center.png[Workday config]
+. In the Workday component, specify *Operation* as `Put_Fund`.
+. Select *Click here to set it up*.
 +
 Configuration example:
 +
 * Configuration Name: `My Workday Connection`
 * Connection: `Basic Authentication Api V32`
-* Username: `my_workday_username`
-* Password: `my_workday_password`
-* Tenant Name: `my_workday_tenant_name`
+* Username: <my_workday_username>
+* Password: <my_workday_password>
+* Tenant Name: <my_workday_tenant_name>
 * Host Name: Defaults to `impl-cc.workday.com`
 +
-. Choose operation *Put_Fund*.
-+
-//image::workday/workday-operation-with-content.png[put fund operation]
-+
-. Add a Logger to the end of your flow that takes a `payload` (or `#[payload]`) as the message.
-. Deploy or run your app.
-. Once the status is `Running`, check for the response.
+. Click *Test* to ensure your user name, password, and tenant name are correct in Workday.
+. Click *Save*.
+. Click *+* and in *Select a component*, select *Logger*.
+. Set *Message* to `payload` (or `#[payload]`).
+. Click *Test* at the right of the taskbar. 
+. After the test is successful, click the down-arrow menu in the taskbar, and select *Deploy application*.
+. After the status is running, check for the response.
```

Comment:
"running" is the status name or a status value; should it be capitalized or bold?